crypto/rsa/rsa_pk1.c

```diff
@@ -236,15 +235,16 @@ int RSA_padding_check_PKCS1_type_2(unsigned char *to, int tlen,
      * should be noted that failure is indistinguishable from normal
      * operation if |tlen| is fixed by protocol.
      */
-    tlen = constant_time_select_int(constant_time_lt(num, tlen), num, tlen);
+    tlen = constant_time_select_int(constant_time_lt(num - 11, tlen),
+                                    num - 11, tlen);
```

Comment:
I don't see why this needs to be done in constant_time, since tlen num-11 is known, and tlen is usually num
but even if it was a secret, it controls the loop below.